src/foundation.cs

```diff
@@ -7378,6 +7378,41 @@ partial interface NSUrlSessionDownloadTask {
 		void Cancel (Action<NSData> resumeCallback);
 	}
 	
+	[Internal]
+	[Static]
+	[NoWatch]
+	partial interface ProxyConfigurationDictionaryKeys {
```

Comment:
is it really `partial` ?
if it is then your `[Static]` and `[Internal]` might overreach

Reply:
Good point, will remove partial.